tests/modules/blockonkeys.c

```diff
@@ -60,6 +60,7 @@ void fsl_free(void *value) {
 /* ========================== helper methods ======================= */
 
 int get_fsl(RedisModuleCtx *ctx, RedisModuleString *keyname, int mode, int create, fsl_t **fsl, int reply_on_failure) {
```

Comment:
i don't understand this function and the "FSL" module, maybe add some doc comment to explain it?